rfc/guides-comments.md

```diff
@@ -0,0 +1,75 @@
+Options available for adding comments to the deployment guides are as follows:
+
+1.  Disqus: <https://disqus.com/>
+
+Really good third party service and the most popular. It has 3 options:
+
+-   Basic: This has ad's (not recommended)
+
+-   Plus $9 per month (Suitable for under 50k visitors daily)
+
+-   Pro $89 per month (Suitable for 150k visitors daily)
+
+Pros:
+
+-   It provides a dashboard to manage for all the comments posted.
+
+-   Analytics available for comments on the dashboard.
+
+-   Very easy to install
+
+Cons
```

Comment:
Two other cons: 

1. [It can add a lot of page weight](https://victorzhou.com/blog/replacing-disqus/). 
1. Since it is JS / AJAX, it's not clear if the content of the comments gets indexed by Google.

Reply:
Yeah, it does add a lot of page weight. About 450kB of data

Reply:
Yea... That is massive.

Note also the comment on JS/AJAX and SEO.

Reply:
Comments actually get indexed by google

Reply:
More info on that?

Reply:
I've also included it in the PR https://code-interactive.com/ad-in/2018/10/does-google-index-disqus-comments/

Reply:
Hm, that's not a lot to go on.

AFAIK, Google can execute _some_ JS and index the contents, but I suspect it doesn't work perfectly, and may not be as effective as server-side rendered HTML. @Etiene Did you come across useful info on this in your research?

Reply:
No, it's not as effective as server-side rendering 
Some discussions concerning this can be found [here](https://www.quora.com/Is-Disqus-actually-SEO-friendly) and [here](https://www.quora.com/Does-Disqus-help-SEO-for-a-website)